templates/security/docker-images.html

```diff
@@ -1,151 +1,272 @@
+          </li>
+          <li class="p-list__item has-bullet">
+            Customer-only content with up to ten years of fixes. <a href="/security/contact-us?product=docker" class="js-invoke-modal">Contact us</a>.
+          </li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>All of our Docker Hub repositories are exempted from per-user rate limits.</p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_2' -%}
+        <h3 class="p-heading--5">
+          Are these Official Images
+          <br class="u-hide--medium u-hide--small" />
+          on Docker Hub?
+        </h3>
+      {%- endif -%}
+      {%- if slot == 'list_item_description_2' -%}
+        <p>
+          Several images from the Canonical LTS Docker Image Portfolio are free Docker Official Image versions during their five year standard security maintenance period. The Ubuntu base image is available both as an official image on Docker hub and through the <a href="https://gallery.ecr.aws/lts/ubuntu">LTS</a> and <a href="https://gallery.ecr.aws/ubuntu/ubuntu">Ubuntu</a> namespaces on Amazon ECR Public.
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_3' -%}
+        <h3 class="p-heading--5">Is the LTS Docker Image Portfolio <br class="u-hide--medium u-hide--small" />a free or a commercial offering?</h3>
+      {%- endif -%}
+      {%- if slot == 'list_item_description_3' -%}
+        <p>
+          Both. Some LTS Docker Images have a free five year maintenance period, based on the underlying Ubuntu LTS free standard security maintenance period. After five years, these LTS images will get five more years of security patches through the Expanded Security Maintenance (ESM) program. The ESM program is available with our Ubuntu Pro subscriptions. Some images don&apos;t get the free five initial LTS years,
+          but still are eligible for the 10-year ESM program. On each image&apos;s documentation, the support dates and LTS/ESM logos indicate the current support status for every version. As with Ubuntu interim releases, ongoing development images are released regularly and receive free security updates while they are the current version.
+        </p>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://canonical.com/blog/canonical-publishes-lts-docker-image-portfolio-on-docker-hub">Read more&nbsp;&rsaquo;</a>
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_4' -%}
+        <h3 class="p-heading--5">Is there a long-term commitment? How long?</h3>
+      {%- endif -%}
+      {%- if slot == 'list_item_description_4' -%}
+        <p>
+          ESM Images are security-maintained for the full ten year period of their underlying Ubuntu LTS release. Some applications will have versions on multiple Ubuntu LTS versions. In each case, the image is maintained for the full life of the underlying Ubuntu LTS.
+        </p>
+      {%- endif -%}
```

Comment:
There is supposed to be a link here as well, please check the copy doc

Reply:
The link opens a modal form instead of directing to another place. Also, the blog is on c.com, not u.com.